suite-native/device-manager/src/components/AddHiddenWalletButton.tsx

```diff
@@ -0,0 +1,27 @@
+import { useSetAtom } from 'jotai';
+
+import { Button } from '@suite-native/atoms';
+import { isPassphraseModalVisibleAtom } from '@suite-native/passphrase';
+import { useTranslate } from '@suite-native/intl';
+
+import { useDeviceManager } from '../hooks/useDeviceManager';
+
+export const AddHiddenWalletButton = () => {
+    const { translate } = useTranslate();
+
+    const setPassphraseModalVisibility = useSetAtom(isPassphraseModalVisibleAtom);
+
+    const { setIsDeviceManagerVisible } = useDeviceManager();
+
+    const handleAddHiddenWallet = () => {
+        setIsDeviceManagerVisible(false);
+        setPassphraseModalVisibility(true);
+        // await dispatch(createDeviceInstance({ device: instance }));
+    };
+
+    return (
+        <Button colorScheme="tertiaryElevation0" onPress={handleAddHiddenWallet}>
```

Comment:
I know you will probably change it later anyway, but the device manager has elevation 1, So the button should have color `tertiaryElevation1`.
```suggestion
        <Button colorScheme="tertiaryElevation1" onPress={handleAddHiddenWallet}>
```

Reply:
That button will probably go, I added it just for testing, but it will most probably be some IconButton at the top in future as it will be modal. Please ignore for now.

Reply:
https://github.com/trezor/trezor-suite/pull/10703/commits/84efcd359621395ff1b61111420ba9d764c6c4f2